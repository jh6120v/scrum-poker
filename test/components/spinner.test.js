import React from 'react';
import { render } from '@testing-library/react';
import Spinner from '../../src/components/spinner';

describe('test <Spinner />', () => {
    it('<Spinner /> snapshot.', () => {
        const { container } = render(<Spinner show />);
        expect(container.firstChild).toMatchSnapshot();
    });
});