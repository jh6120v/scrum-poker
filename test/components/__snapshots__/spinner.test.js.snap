// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <Spinner /> <Spinner /> snapshot. 1`] = `
<div
  class="sc-bdVaJa cmqDfb"
>
  <div
    class="sc-bwzfXH bggjYU"
  >
    <div
      class="sc-htpNat fntENR"
    />
    <div
      class="sc-htpNat fntENR"
    />
    <div
      class="sc-htpNat fntENR"
    />
  </div>
</div>
`;
